Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.h

```diff
@@ -59,17 +59,10 @@ class CDMInstanceOpenCDM final : public CDMInstance {
     CDMInstanceOpenCDM(const CDMInstanceOpenCDM&) = delete;
     CDMInstanceOpenCDM& operator=(const CDMInstanceOpenCDM&) = delete;
 
-    class Session {
-    private:
-        Session() = delete;
-        Session(const Session&) = delete;
-        Session& operator=(const Session&) = delete;
-
+    class Session : public ThreadSafeRefCounted<Session> {
     public:
-        Session(const media::OpenCdm& source, Ref<WebCore::SharedBuffer>&& initData);
-        ~Session() = default;
+        static Ref<Session> create(const media::OpenCdm& source, Ref<WebCore::SharedBuffer>&& initData);
```

Comment:
This is just a comment, not a request: something in my roadmap is that I want to change the `Ref<WebCore::SharedBuffer>` into an `InitData`.

Reply:
Agreed, we have a little bit of inconsistency with the existing InitData which is aliased to a String rather than a SharedBuffer, can be addressed in a later cleanup :)